src/infrastructure/services/creditCard/CreditCardService.ts

```diff
@@ -29,13 +49,22 @@ const creditCardApi = api.injectEndpoints({
         method: 'DELETE',
       }),
     }),
+    getPublicKeys: builder.mutation<PublicKeysResult, void>({
```

Comment:
WDYT about removing this as it is no needed anymore?

Reply:
Yes, you're right. Already removed!